vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -98,6 +98,9 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
+      if new_result[:ems_ref] == nil # Tipically this happens for kubernetes services
```

Comment:
@abonas correct `"kubernetes"` and `"kubernetes-ro"` seem not to have an `uid` anymore.